### DISCUSSION TITLE
Integrate Picmo with Wix/Editor X site

### DISCUSSION THREAD

**Discussion by rasmushauschild** [joeattardi/picmo#277] [Q&A]
Hi Joe!
Thank you so much for putting together Picmo!

I'm building a web app using Editor X from Wix, and I'm having trouble getting a Picmo emoji picker to show up.
I have created an empty container `box149`, and I'm running the following code when the site loads:
```
import { createPicker } from 'picmo'';

$w.onReady(function () {
const container = $w("#box149");
const picker = createPicker({ rootElement : container});
}
```

Through a little debugging, I can tell that the code execution freezes at this point:
`const picker = createPicker({ rootElement : container});`

Do you have any idea what might be going wrong, or something I can try out?

Thank you in advance!
Rasmus

**Comment by joeattardi** [comment]
I'm not familiar with Wix or Editor X, when you say the execution freezes do you mean it freezes the browser?
